tests/test_ramp_fitting.py

```diff
@@ -0,0 +1,246 @@
+import numpy as np
+
+from stcal.ramp_fitting.ramp_fit import ramp_fit_data
+from stcal.ramp_fitting.ramp_fit_class import RampData
+
+DELIM = "-" * 70
+
+# single group intergrations fail in the GLS fitting
+# so, keep the two method test separate and mark GLS test as
+# expected to fail.  Needs fixing, but the fix is not clear
+# to me. [KDG - 19 Dec 2018]
+
+dqflags = {
+    'DO_NOT_USE':       2**0,   # Bad pixel. Do not use.
+    'SATURATED':        2**1,   # Pixel saturated during exposure
+    'JUMP_DET':         2**2,   # Jump detected during exposure
+    'NO_GAIN_VALUE':    2**19,  # Gain cannot be measured
+    'UNRELIABLE_SLOPE': 2**24,  # Slope variance large (i.e., noisy pixel)
+}
+
+
+# -----------------------------------------------------------------------------
+#                           Test Suite
+
+
+def test_utils_dq_compress_final():
+    """
+    Set up a multi-integration 3 pixel data array each ramp as the following:
```

Comment:
I don't follow what is being tested here - can you add some clarification ?

Reply:
I am expanding the comment for this test.  Let me know if I need to expand it further.